load_tests/load_test.py

```diff
@@ -54,8 +54,8 @@ def get_log_delay(log_delay_epoch_time):
 # Set buffer for waiting all logs sent to destinations (~5min)
 def set_buffer(stop_epoch_time):
     curr_epoch_time = time.time()
-    if curr_epoch_time-stop_epoch_time < BUFFER_TIME_IN_SECOND:
-        time.sleep(int(BUFFER_TIME_IN_SECOND-curr_epoch_time+stop_epoch_time))
+    if (curr_epoch_time - stop_epoch_time) < BUFFER_TIME_IN_SECOND:
```

Comment:
do we need brackets ?

Reply:
I checked and the math does work

Reply:
By brackets do you mean `{`?? Python doesn't use those